StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ExternalPaymentOption.swift

```diff
@@ -29,15 +29,21 @@ class ExternalPaymentOption {
     /// URL of a 48x pixel, variable width tall PNG representing the payment method suitable for display against a dark background color. If `nil`, use `lightImageUrl` instead.
     let darkImageUrl: URL?
 
+    /// When false, Mobile Payment Element will collect billing details for this custom payment method type
+    /// in accordance with the `billingDetailsCollectionConfiguration` settings.
+    /// This has no effect if `billingDetailsCollectionConfiguration` is not configured.
```

Comment:
`not configured` seems vague, since the default is `.auto`  -- though i realize this is a copy and paste from custompayment methods.

Reply:
Yeah just copied over, gonna leave as-is